src/include/ipc/info.h

```diff
@@ -122,7 +122,7 @@ struct sof_ipc_cc_version {
 
 	char name[16]; /* null terminated compiler name */
 	char optim[4]; /* null terminated compiler -O flag value */
-	char desc[]; /* null terminated compiler description */
+	char desc[24]; /* null terminated compiler description */
```

Comment:
We still need to align the PR with the kernel side.

Reply:
no longer aligned with latest PR.

Reply:
I hope this corresponding PR will be soon merge, then this cherry-pick will be deleted.